docs/.vuepress/config.js

```diff
@@ -1,134 +0,0 @@
-module.exports = {
-  theme: "cosmos",
-  title: "Ignite CLI",
-  head: [
-    [
-      "script",
-      {
-        async: true,
-        src: "https://www.googletagmanager.com/gtag/js?id=G-XL9GNV1KHW",
```

Comment:
Do we have analytics enabled in the new template?

Reply:
good question @ilgooz 
I don't know much about analytics in the template... is there action required from my end?

Reply:
Not yet. But thank you for the reminder!

Reply:
Done 👍